docs/infrastructure/microk8s-quickstart.md

```diff
@@ -12,12 +12,12 @@ Follow the instructions below to set up a MicroK8s instance from scratch, includ
 
 Prepare your environment to set up the MicroK8s instance.
 
--   You will need a machine running [Ubuntu Server 18.04 LTS](https://ubuntu.com/download/server) that:
+-   You will need a machine running [Ubuntu Server 20.04 LTS](https://ubuntu.com/download/server) that:
```

Comment:
Taking the chance to update this to a more recent LTS version. I setup a testing environment with 20.04 and it went fine. Is there reason why we shouldn't do this?